### PR TITLE
ET-3509 deploy containers

### PR DESCRIPTION
**Pull request by Robert-Steiner**
## What 🕵️ 🔍

- renames `user` and `document` to `users` and `documents`
- adds environment variables
- removes hello_world task definition & service
- creates target groups for each service
- adds alb path rules

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/ET-3509)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

**Comment by Robert-Steiner** [comment]
the postgres connection works. with the elasticsearch we still have issues but it can already be reviewed while we investigate

File: aws/stage/vpc_endpoint/main.tf [hunk 1]
```diff
@@ -64,5 +64,6 @@ resource "aws_ssm_parameter" "elasticsearch_dns" {
   description = "The Elasticsearch DNS entry"
   name        = "/elasticsearch/dns"
   type        = "String"
-  value       = aws_vpc_endpoint.elasticsearch.dns_entry[0]["dns_name"]
+  # TODO check if we can get this value from somewhere else
+  value = "https://elasticsearch-staging.es.vpce.eu-central-1.aws.elastic-cloud.com"
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply aws/stage/vpc_endpoint/main.tf:68]: done

[the remark this answers is not in the document]
Comment (jorge-cr-13) [reply aws/stage/vpc_endpoint/main.tf:68]: Do you still need this resource?

[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply aws/stage/vpc_endpoint/main.tf]: Good catch! i thought i removed it

**Review by jorge-cr-13 — DISMISSED** [review]

**Review by jorge-cr-13 — APPROVED** [review]

**Pull request merged**